syntax = "proto3";

import "google/protobuf/empty.proto";

package refresh;

message RefreshRequest {
    string user_id = 1;
    string token = 2;
}

service RefreshService {
    rpc RefreshToken (RefreshRequest) returns (google.protobuf.Empty);
}
